api/src/opentrons/protocol_api/labware.py

```diff
@@ -326,9 +326,11 @@ def load_liquid(self, liquid: Liquid, volume: float) -> None:
         :param Liquid liquid: The liquid to load into the well.
         :param float volume: The volume of liquid to load, in µL.
 
-        .. TODO: flag as deprecated in 2.22 docs
-            In API version 2.22 and later, use :py:meth:`~Labware.load_liquid`, :py:meth:`~Labware.load_liquid_by_well`,
-            or :py:meth:`~Labware.load_empty` to load liquid into a well.
+        .. deprecated:: 2.23
+
+        In API version 2.22 and later, use :py:meth:`.Labware.load_liquid`, :py:meth:`.Labware.load_liquid_by_well`,
+            or :py:meth:`.Labware.load_empty` to load liquid into a well.
+
```

Comment:
Sneaky but this should be 2.22.
```suggestion
        .. deprecated:: 2.22

        Use :py:meth:`.Labware.load_liquid`, :py:meth:`.Labware.load_liquid_by_well`,
            or :py:meth:`.Labware.load_empty` instead.

```